mkdocs/docs/HPC/torque_frontend_via_jobcli.md

```diff
@@ -0,0 +1,153 @@
+
+## What is Torque
+
+[Torque](https://en.wikipedia.org/wiki/TORQUE) is a resource manager for submitting and managing jobs on an HPC cluster. It is an implementation of [PBS (Portable Batch System)](https://en.wikipedia.org/wiki/Portable_Batch_System).
+Torque is not widely used anymore, so the {{hpcinfra}} no longer uses Torque in the backend since 2021 in favor of Slurm.
+The Torque user interface, which consists of commands like `qsub` and `qstat`, were kept however, to avoid that researchers had to learn other commands to submit and manage jobs.
+
+## Slurm backend
+
+[Slurm](https://en.wikipedia.org/wiki/Slurm_Workload_Manager) is a resource manager for submitting and managing jobs on an HPC cluster, similar to Torque (but more advanced/modern in some ways). Currently, Slurm is the most popular workload manager on HPC systems worldwide, but it has a user interface that is different and in some sense less user friendly than Torque/PBS.
+
+## jobcli
+
+Jobcli is a Python library that was developed by {{hpcteam}} to make it possible for the {{hpcinfra}} to use a Torque frontend and a Slurm backend. In addition to that, it adds some additional options for Torque commands. Put simply, jobcli can be thought of as a Python script that "translates" Torque commands into equivalent Slurm commands, and in the case of `qsub` also makes some changes to the provided job script to make it compatible with Slurm.
+
+### Additional options for Torque commands supported by jobcli
+
+#### help option
+
+Adding `--help` to a Torque command when using it on the {{hpcinfra}} will output an extensive overview of all supported options for that command, including all possible options for that command (including the original ones from Torque and the ones added by jobcli) and a short description for each one.
+
+For example:
+```shell
+$ qsub --help
+usage: qsub [--version] [--debug] [--dryrun] [--pass OPTIONS] [--dump PATH]...
+
+Submit job script
+
+positional arguments:
+  script_file_path      Path to job script to be submitted (default: read job
+                        script from stdin)
+
+optional arguments:
+  -A ACCOUNT            Charge resources used by this job to specified account
+  ...
+```
+
+#### dryrun option
+
+Adding `--dryrun` to a Torque command when using it on the {{hpcinfra}} will show the user what Slurm commands are generated by that Torque command by jobcli. Using `--dryrun` will not actually execute the Slurm backend command.
+
+See [the examples](./#examples)
+
+#### debug option
+
+Similarly to `--dryrun`, adding `--debug` to a Torque command when using it on the {{hpcinfra}} will show the user what Slurm commands are generated by that Torque command by jobcli. However in contrast to `--dryrun`, using `--debug` will actually submit the job afterwards.
+
+See [the examples](./#examples)
+
+#### Examples
+
+The following examples illustrate the working of the `--dryrun` and `--debug` options with and example jobscript.
```

Comment:
The following examples illustrate the working of the `--dryrun` and `--debug` options with an example jobscript.